### PR TITLE
tweaks some evil fucking punctuation

### PR DESCRIPTION
**Pull request by PKPenguin321**
jesus fuck this thing annoyed me every single god damn time i saw it and i am so glad i am finally removing it

### CODE REVIEW

**Comment by Aloraydrel** [comment]
this......................
..........
..........
...........

YET

**Comment by Archie700** [comment]
PRIORITY: CRITICAL

**Comment by Xhuis** [comment]
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -355,8 +355,8 @@ var/list/admin_verbs_hideable = list(
 		src << "<font color='red'>Error: Aghost: Can't admin-ghost whilst in the lobby. Join or Observe first.</font>"
 	else
 		//ghostize
-		log_admin("[key_name(usr)] admin ghosted")
-		message_admins("[key_name_admin(usr)] admin ghosted")
+		log_admin("[key_name(usr)] admin ghosted.")
```
Comment (Cheridan) [inline code/modules/admin/admin_verbs.dm]: cheribucks obtained

**Pull request merged**